### ISSUE TITLE
app specific functionality [enhancement]

### ISSUE THREAD

**Issue by deluxeroyale** [noah-nuebling/mac-mouse-fix#650]
## Description
Add a feature that allows the user to apply specific "profiles" for different apps. 
Photoshop profile for example or Cinema 4D profile.
And of course the functionality of enabling and disabling the smooth scroll. (like 2.0)

## Rationale
In photoshop and illustrator I use mouse keyboard shortcuts added to mouse specific buttons. These are different depending on the app. Having one universal "profile" makes things break the way I use the mouse over different applications.
Basically enabling app specific settings makes the app better without bloating as I see this implemented with a dropdown listing apps.

## Additional info
–

**Comment by janzen0816** [comment]
I also think this is a much-needed feature to add right now (it was already there in 2.0). 
Most of the current issues are also about the call for this feature to return.
I hope Noah will add this much needed feature in the next update. 
Best，
